File: Backend/TrackrAPI/TrackrAPI/Dtos/Catalogo/DominioHospitalDto.cs
namespace TrackrAPI.Dtos.Catalogo;

public class DominioHospitalDto
{
    public int? IdDominioHospital {  get; set; }
    public int IdDominio {  get; set; }
    public int IdHospital { get; set; }
    public int? LongitudMinima { get; set; }
    public int? LongitudMaxima { get; set; }
    public decimal? ValorMinimo { get; set; }
    public decimal? ValorMaximo { get; set; }
    public DateTime? FechaMinima { get; set; }
    public DateTime? FechaMaxima { get; set; }
    public bool ? PermiteFueraDeRango { get; set; }
    public string UnidadMedida { get; set; }
}

File: Frontend/trackr-administrador/src/app/shared/dtos/catalogo/dominio-hospital-dto.ts
export interface DominioHospitalDto{
    idDominioHospital?:number;
    idDominio:number;
    idHospital:number;
    longitudMinima?: number;
    longitudMaxima?: number;
    valorMinimo?: number;
    valorMaximo?: number;
    fechaMinima?: Date;
    fechaMaxima?: Date;
    permiteFueraDeRango?: boolean;
    unidadMedida?: string;
}